libtorch/Dockerfile

```
@@ -42,7 +42,7 @@ ENV CUDA_HOME /usr/local/cuda
 FROM base as conda
 ADD ./common/install_conda.sh install_conda.sh
 RUN bash ./install_conda.sh && rm install_conda.sh
-RUN /opt/conda/bin/conda install -y cmake=3.14
+RUN /opt/conda/bin/conda install -y cmake=3.22.3
```

Let's stick with 3.14 as it allows us to validate that CMakeFile is correct at minspec
```suggestion
RUN /opt/conda/bin/conda install -y cmake=3.14
```